doc/classes/OS.xml

```diff
@@ -408,6 +408,15 @@
 				[b]Note:[/b] On Web platforms, it is still possible to determine the host platform's OS with feature tags. See [method has_feature].
 			</description>
 		</method>
+		<method name="get_process_exit_code" qualifiers="const">
+			<return type="int" />
+			<param index="0" name="pid" type="int" />
+			<description>
+				Returns the exit code of a spawned process once it has finished running (see [method is_process_running]).
+				Returns [code]-1[/code] if the [param pid] is not a PID of a spawned child process, the process is still running or the method is not implemented for the current platform.
+				[b]Note:[/b] This method is implemented on Linux and Windows.
```

Comment:
It should work on macOS and Android as well, since both inherit generic `OS_Unix` code.

Reply:
Oh that's why I couldn't find other methods.